File: app/src/main/java/com/wealoha/social/beans/remotelog/ClientLogService.java
package com.wealoha.social.beans.remotelog;

import retrofit.http.Field;
import retrofit.http.FormUrlEncoded;
import retrofit.http.POST;

import com.wealoha.social.beans.Result;
import com.wealoha.social.beans.ResultData;
import com.wealoha.social.impl.ServerUrlImpl;

/**
 * 
 * @author hongwei
 * @createTime Jan 12, 2015 12:11:30 PM
 */
public interface ClientLogService {

	@POST(ServerUrlImpl.CLIENT_LOG)
	@FormUrlEncoded
	public Result<ResultData> log(@Field("message") String message, @Field("exception") String exception, //
			@Field("timestamp") Long createTime);

}

File: app/src/main/java/com/wealoha/social/api/ClientLogService.kt
package com.wealoha.social.api

import com.wealoha.social.beans.Result
import com.wealoha.social.beans.ResultData
import com.wealoha.social.impl.ServerUrlImpl
import retrofit.http.Field
import retrofit.http.FormUrlEncoded
import retrofit.http.POST

/**
 *
 * @author hongwei
 * @createTime Jan 12, 2015 12:11:30 PM
 */
interface ClientLogService {
    @POST(ServerUrlImpl.CLIENT_LOG)
    @FormUrlEncoded
    fun log(
        @Field("message") message: String?,
        @Field("exception") exception: String?,  //
        @Field("timestamp") createTime: Long?
    ): Result<ResultData?>?
}